src/client/pages/PlaceList/PlaceList.jsx

```diff
@@ -93,7 +93,7 @@ class PlaceList extends React.Component {
         this.setState({
           name: name,
           username: username,
-          profilePictureLink: profile_photo ? profile_photo : '../../../../assets/common/user-icon.png',
+          profilePictureLink: profile_photo
```

Comment:
u need to change the render also otherwise they won't load the default icon right?

Reply:
It's automatically handled by the ProfileButton. If profilePictureLink is null it will load the default icon